src/paste-markdown-image-link.js

```diff
@@ -21,15 +21,23 @@ function onDrop(event: DragEvent) {
   if (hasFile(transfer)) return
   if (!hasLink(transfer)) return
 
+  let linkified = false
+  const transformedLinks = links(transfer).map(link => {
+    if (isImageLink(link)) {
+      linkified = true
+      return `\n![](${link})\n`
+    } else {
+      return link
+    }
+  })
+  if (!linkified) return
+
```

Comment:
To avoid maintaining state variables outside the loop, and reduce a little duplication at the next call site, we can test with `Array#some`.

```js
if (!links(transfer).some(isImageLink)) return
const transformedLinks = links(transfer).map(linkify)
```